controllers/util/pvcs_util.go

```diff
@@ -84,17 +95,18 @@ func ListPVCsByPVCSelector(
 func IsPVCInUseByPod(ctx context.Context,
 	k8sClient client.Client,
 	log logr.Logger,
-	pvcName, pvcNamespace string,
+	pvcNamespacedName types.NamespacedName,
 	inUsePodMustBeReady bool,
 ) (bool, error) {
+	log = log.WithValues("pvc", pvcNamespacedName.String())
 	podUsingPVCList := &corev1.PodList{}
 
 	err := k8sClient.List(ctx,
 		podUsingPVCList, // Our custom index - needs to be setup in the cache (see IndexFieldsForVSHandler())
-		client.MatchingFields{PodVolumePVCClaimIndexName: pvcName},
-		client.InNamespace(pvcNamespace))
+		client.MatchingFields{PodVolumePVCClaimIndexName: pvcNamespacedName.Name},
+		client.InNamespace(pvcNamespacedName.Namespace))
```

Comment:
We have a problem here. `pvcNamespacedName.Namespace` is empty because it wasn't filled in at the source.